src/client/sky.cpp

```diff
@@ -700,69 +570,198 @@ void Sky::update(float time_of_day, float time_brightness,
 
 			video::SColorf pointcolor_sun_f(1, 1, 1, 1);
 			if (m_sun_tonemap) {
-				pointcolor_sun_f.r = pointcolor_light *
-					(float)m_materials[3].EmissiveColor.getRed() / 255;
-				pointcolor_sun_f.b = pointcolor_light *
-					(float)m_materials[3].EmissiveColor.getBlue() / 255;
+				pointcolor_sun_f.r =
+						pointcolor_light *
+						(float)m_materials[3]
+								.EmissiveColor.getRed() /
+						255;
+				pointcolor_sun_f.b =
+						pointcolor_light *
+						(float)m_materials[3]
+								.EmissiveColor.getBlue() /
+						255;
 				pointcolor_sun_f.g = pointcolor_light *
-					(float)m_materials[3].EmissiveColor.getGreen() / 255;
+						     (float)m_materials[3]
+								     .EmissiveColor
+								     .getGreen() /
+						     255;
 			} else {
 				pointcolor_sun_f.r = pointcolor_light * 1;
-				pointcolor_sun_f.b = pointcolor_light *
-					(0.25 + (rangelim(m_time_brightness, 0.25, 0.75) - 0.25) * 2 * 0.75);
-				pointcolor_sun_f.g = pointcolor_light * (pointcolor_sun_f.b * 0.375 +
-					(rangelim(m_time_brightness, 0.05, 0.15) - 0.05) * 10 * 0.625);
+				pointcolor_sun_f.b =
+						pointcolor_light *
+						(0.25 + (rangelim(m_time_brightness, 0.25,
+									 0.75) -
+									0.25) * 2 *
+										0.75);
+				pointcolor_sun_f.g =
+						pointcolor_light *
+						(pointcolor_sun_f.b * 0.375 +
+								(rangelim(m_time_brightness,
+										 0.05,
+										 0.15) -
+										0.05) *
+										10 *
+										0.625);
```

Comment:
Thanks LINT/Clang format. This looks really awful. The previous formatting in this function was way easier to read IMO.

Reply:
Please revert this bad suggestion by clang format.